src/Controller/ChoicesController.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\ChoiceManager;
+use App\Model\QuestionManager;
+
+/**
+ * Class ChoicesController
+ * Uses ChoiceManager
+ * Creates page that allows user to add their own choices
+ * Creates page that tells the user if their question/choices have successfully been added
+ */
+class ChoicesController extends SessionController
+{
+    public const DATABASE_ERROR = -1;
+
+    /**
+     * Creates the page that allows users to enter their choices
+     * @return string
+     */
+    public function add(): string
+    {
+        // Redirect to question/add if it has not been filled correctly
+        if (!isset($_SESSION['userQuestion']) || !isset($_SESSION['numberOfChoices'])) {
+            header('Location: ../question/add');
+            return "";
+        }
+
+        $errors = [];
+
+        if (isset($_POST['userChoices'])) {
+            // Checks on userChoices
+            $choiceManager = new ChoiceManager();
+            $errors = $choiceManager->choicesVerifications($_POST['userChoices']);
+
+            // Display errors if there are any
+            if (!empty($errors)) {
+                return $this->twig->render(
+                    'Choices/add.html.twig',
+                    ['userQuestion' => $_SESSION['userQuestion'] ,
+                    'numberOfChoices' => $_SESSION['numberOfChoices'] ,
+                    'errors' => $errors]
+                );
+            }
+
+            // Put choices into $_SESSION
+            if (isset($_SESSION['userChoices'])) {
+                unset($_SESSION['userChoices']);
+            }
```

Comment:
Seems useless because you'll replace $_SESSION['userChoices'] value just afterwards